data/json/mutations/mutations.json

```diff
@@ -4008,7 +4008,7 @@
     "type": "mutation",
     "id": "HORNS_CURLED",
     "name": { "str": "Curled Horns" },
-    "points": -3,
+    "points": 0,
```

Comment:
horns and curled horns both need to restrict gear on the head

Reply:
Originally it was regular horns allow all headgear, curled horns allow no headgear and pointed horns/antennae/antlers allow soft headgear only, but I went ahead and made regular horns soft headgear only.